src/slot.rs

```diff
@@ -28,7 +28,7 @@ impl Slot {
 
     /// Returns an infinite iterator that yields the first slot in each window.
     pub fn windows() -> impl Iterator<Item = Self> {
-        (0..).map(Self)
```

Comment:
ouch, that is such a silly bug.  I really should've written some tests when I wrote this code 😢